python/dataproc_templates/hbase/README.md

```diff
@@ -88,12 +91,15 @@ optional arguments:
 ```
 
 ## Example submission
-
+**Manual Process**
 ```
+
 export GCP_PROJECT=<project_id>
 export SUBNET=<subnet>
 export REGION=<region>
-export GCS_STAGING_LOCATION=<gcs-staging-bucket-folder> 
+export GCS_STAGING_LOCATION=<gcs-staging-bucket-folder>
+
+#For manual process JARS environment variable need to be set. Not required for automated process
 export JARS="gs://<your_bucket_to_store_dependencies>/hbase-client-2.4.12.jar, \
```

Comment:
can we add htrace-core4-4.2.0-incubating.jar dep in the manual process section as well?

Reply:
Thanks for the suggestion. Since, this jar is specific to few of the Hbase version, early- 2.x , hence, this is captured in the comment section. I can add an optional jar export example for htrace-core4-4.2.0-incubating.jar.

Reply:
yes that would be great